debian/rules

```diff
@@ -4,7 +4,5 @@ export DH_OPTIONS
 export DH_GOPKG := github.com/cloudflare/cfssl_trust/cmd/cfssl-trust
 
 %:
-	# go get github.com/Masterminds/glide
-	# glide i
```

Comment:
I'm not familiar with debian packaging. Are these safe to remove?

Reply:
Yep, this was an early attempt at debian packaging that maybe shouldn't have been in the repo yet.